content/post/kubevault-webinar-2023.01.17/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Manage Redis Secrets using KubeVault
+date: "2023-01-18"
+weight: 20
+authors:
+- Abdullah Al Shaad
+tags:
+- cli
+- hashicorp
+- kubernetes
+- kubevault
+- kubevault-cli
+- redis
+- secret-management
+- security
+- vault
+---
+
+## Summary
+
+AppsCode held a webinar on **"Manage Redis Secrets using KubeVault"**. This took place on 17th Jan 2023. The contents of what took place at the webinar are shown below:
+- Deploy VaultServer
+- Deploy Redis Standalone Database
+- Enable Redis SecretEngine
+- Create Database Roles
+- Manage User Privileges
+- KubeVault CLI in Action
+- Q & A Session
+
+## Description of the Webinar
+
+It is required to install the followings to get started:
+- KubeDB Enterprise Operator
+- KubeVault Enterprise Operator
+- Secrets Store CSI Driver
+- Vault Specific CSI Provider
+
+The speaker starts by deploying `VaultServer` & `Redis` Database by `KubeDB`. Speaker shows how easy it is to get the decrypted `vault-root-token` using KubeVault CLI. Followed by, enabling `SecretEngine` & creating some Database `Roles`.
```

Comment:
may change to passive: Speaker shows > Later it was shown,

Reply:
"It was shown" , in the next sentence